nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -618,7 +617,6 @@ in
           restartTriggers = [ config.environment.etc."ssh/sshd_config".source ];
 
           serviceConfig = {
-            Type = "notify";
```

Comment:
I'm pretty sure this should work fine, but it's also fine for me to re-introduce this later.

Reply:
@NyCodeGHG Yea, I'm pretty sure you're right that non-socket-activated `sshd.service` now supports `sd_notify`, which it didn't in the past. But I want to introduce that improvement in a separate PR where we can more seriously test and check it.